pkg/util/kernel/find_headers.go

```diff
@@ -71,9 +74,18 @@ func getHeaderDirs(v Version) ([]string, error) {
 	}
 
 	// KernelVersion == uname -r
-	dirs := []string{fmt.Sprintf(kernelModulesPath, hi.KernelVersion)}
+	// check KernelHeadersDownloadDir to see if we've previously downloaded kernel headers
+	dirs := []string{
+		fmt.Sprintf(kernelModulesPath, hi.KernelVersion),
+		fmt.Sprintf(KernelHeadersDownloadDir+kernelModulesPath, hi.KernelVersion),
```

Comment:
Combining directories should use `filepath.Join` to prevent problems with separators.